apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/RdaClaimMessageMetaData.java

```diff
@@ -87,6 +87,22 @@ public class RdaClaimMessageMetaData {
   @Column(name = "received_date")
   private Instant receivedDate;
 
+  /** Identifies the particular phase associated with this claim (1, 2, 3) */
+  @Column(name = "phase")
+  private Short phase;
+
+  /** Identifies the particular sequence number of the phase associated with this claim (0, 1, 2) */
```

Comment:
```suggestion
  /** Identifies the particular sequence number of the phase associated with this claim (0, 1, 2, N) */
```
(I've seen numbers in the teens so far, so not sure what the limit is here)